### PR TITLE
fix(ContactsCollection): Check selector to prevent undefined

### PR DESCRIPTION
**Pull request by cballevre**
This problem appears when you only have partialIndex so the selector which is where return undefined. This problem was found into : https://github.com/cozy/cozy-contacts/pull/898

### CODE REVIEW

**Review by Crash-- — COMMENTED** [review]
Can you add a test please?

File: packages/cozy-stack-client/src/ContactsCollection.spec.js [hunk 1]
```diff
@@ -36,6 +36,16 @@ describe('ContactsCollection', () => {
     jest.spyOn(console, 'warn').mockImplementation(() => {})
   })
 
+  it('should handle undefined selector', async () => {
+    const { col, stackClient } = setup()
+    await col.find()
+    expect(stackClient.fetchJSON).toHaveBeenCalledWith(
+      'POST',
+      '/data/io.cozy.contacts/_find',
+      expect.objectContaining({ selector: undefined })
```
Comment (paultranvan) [inline packages/cozy-stack-client/src/ContactsCollection.spec.js:45]: Does that mean we actually send `selector: undefined` to the stack, or the selector attribute is not there?

Comment (cballevre) [reply packages/cozy-stack-client/src/ContactsCollection.spec.js:45]: not if we have an partial index with combine with selector. I didn't define partialFilter for this test so it's return undefined

Comment (Crash--) [reply packages/cozy-stack-client/src/ContactsCollection.spec.js:45]: The test should be : 
`it('call find route if there is no selector')`

and we should just test that `_find` has been called. We don't need to test the object

**Comment by Crash--** [comment]
ping @cballevre

**Review by Crash-- — APPROVED** [review]

**Pull request merged**